<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="1" skipped="0" tests="12" time="0.141" timestamp="2024-12-29T14:23:53.847091+00:00" hostname="9338220b6496"><testcase classname="tests.test_calculator" name="test_add" time="0.001"><failure message="assert 5 == 6&#10; +  where 5 = add(2, 3)">def test_add():
        # 同値分割の例: 正数、負数、0
&gt;       assert add(2, 3) == 6
E       assert 5 == 6
E        +  where 5 = add(2, 3)

tests/test_calculator.py:7: AssertionError</failure></testcase><testcase classname="tests.test_calculator" name="test_subtract" time="0.000" /><testcase classname="tests.test_calculator" name="test_divide" time="0.000" /><testcase classname="tests.test_discount_logic" name="test_discount_rate[gold-True-5000-20]" time="0.000" /><testcase classname="tests.test_discount_logic" name="test_discount_rate[gold-True-9999-20]" time="0.000" /><testcase classname="tests.test_discount_logic" name="test_discount_rate[gold-True-4999-15]" time="0.000" /><testcase classname="tests.test_discount_logic" name="test_discount_rate[gold-False-5000-10]" time="0.000" /><testcase classname="tests.test_discount_logic" name="test_discount_rate[silver-True-3000-10]" time="0.000" /><testcase classname="tests.test_discount_logic" name="test_discount_rate[silver-False-3000-5]" time="0.000" /><testcase classname="tests.test_discount_logic" name="test_discount_rate[normal-True-5000-5]" time="0.000" /><testcase classname="tests.test_discount_logic" name="test_discount_rate[normal-False-4999-0]" time="0.000" /><testcase classname="tests.test_state_machine" name="test_state_machine" time="0.000" /></testsuite></testsuites>